src/nanopolish_polya_estimator.cpp

```diff
@@ -333,9 +342,9 @@ class SegmentationHMM {
     // ----- for a given sample value and shift/scale parameters, return log-probs for each state:
     std::vector<float> log_probas(const float x) const
     {
-	std::vector<float> log_proba(6);
+        std::vector<float> log_proba(6);
         for (uint8_t k = 0; k < 6; ++k) {
```

Comment:
I suggest having a final enum HMM_NUM_STATES instead of a hardcoded value here

Reply:
Updated with another enum label.